test/functional/mining_basic.py

```
@@ -71,6 +74,45 @@
+    def test_blockmintxfee_parameter(self) -> None:
+        self.log.info("Test -blockmintxfee setting")
+        self.restart_node(0, extra_args=['-minrelaytxfee=0', '-persistmempool=0'])
+        node = self.nodes[0]
+
+        # test default (no parameter), zero and a bunch of arbitrary blockmintxfee rates [sat/kvB]
+        for blockmintxfee_sat_kvb in (DEFAULT_BLOCK_MIN_TX_FEE, 0, 50, 100, 500, 2500, 5000, 21000, 333333, 2500000):
+            blockmintxfee_btc_kvb = blockmintxfee_sat_kvb / Decimal(COIN)
+            if blockmintxfee_sat_kvb == DEFAULT_BLOCK_MIN_TX_FEE:
+                self.log.info("-> Default -blockmintxfee setting (%d sat/kvB)...", blockmintxfee_sat_kvb)
+            else:
+                blockmintxfee_parameter = "-blockmintxfee={:.8f}".format(blockmintxfee_btc_kvb)
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)